OPERATOR.md

```diff
@@ -30,19 +30,17 @@ These features are exclusively provided by this enterprise version:
 - Kafka deployment including the corresponding Zookeeper
 - Openwhisk Invoker fully integrated with Zookeeper and Kafka
 - Openwhisk Controller fully integrated with Kafka
-- Possibility to deploy Openshisk Controller in lean mode
+- Possibility to deploy Openwhisk Controller in lean mode
 
 ## Publishing the operator
+
 Once the operator is ready, you can build and test it a against a kubernetes cluster.
 First, generate a new image tag with `task image-tag`.
-You can test locally using the kind cluster (provided by default by the development environment) with  `ent:build-and-load`. 
-
+You can test locally using the kind cluster (provided by default by the development environment) with `ent:build-and-load`.
```

Comment:
Actually all the ent:* related tasks have to be removed. OpenServerless embeds the original enterprise operator, so it is possible to use the `build-and-load` and `build-and-push` tasks.

Reply:
I changed the references to `task ent:*`. It should be  ok now